src/utils.ts

```diff
@@ -85,8 +85,8 @@ export function waitForSessionFile(sessionFilePath: string, callback: IWaitForSe
         }
     }
 
-    // Try once per second for 60 seconds, one full minute
-    innerTryFunc(60, 1000);
+    // Try once every 2 seconds for 120 seconds, two full minutes
+    innerTryFunc(120, 2000);
```

Comment:
Doesn't this bump the max wait time to 4 mins?  2 * 120 secs.  Maybe just use `(120, 1000)`?

Reply:
Good point. I'm thinking it should be 60, 2000 maybe? Ideally this should be exponential backoff style perhaps?